1/Dockerfile

```diff
@@ -32,20 +32,22 @@ EXPOSE 8080 50000
 
 RUN curl http://pkg.jenkins-ci.org/redhat/jenkins.repo -o /etc/yum.repos.d/jenkins.repo && \
     rpm --import http://pkg.jenkins-ci.org/redhat/jenkins-ci.org.key && \
-    yum install -y tar zip unzip java-1.8.0-openjdk jenkins-1.609 && \
+    rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
+    yum -y --setopt=tsflags=nodocs --enablerepo=centosplus install epel-release && \
+    rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7 && \
+    yum -y --setopt=tsflags=nodocs install gettext git zip unzip java-1.8.0-openjdk jenkins-1.609 nss_wrapper && \
```

Comment:
@bparees I sneaked `git` here as I think it will be common dependency if you want to use GIT repository in Jenkins job... this reduce the time needed to build jenkins-master ;-)